zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/repo/AzureNotebookRepo.java

```diff
@@ -244,4 +245,11 @@ public Note setNoteRevision(String noteId, String revId, AuthenticationInfo subj
     // Auto-generated method stub
     return null;
   }
+
+  @Override
+  public FileInfo rename(FileInfo oldFile, FileInfo newFile, AuthenticationInfo subject)
```

Comment:
Sorry, I don't think it is time to add rename method interface. Because rename should be called when note title is changed. But for now I think title renaming should not trigger any operation in NotebookRepo. So I think it is better to do the renaming in save method.